sweepbatcher/sweep_batch.go

```diff
@@ -700,9 +701,11 @@ func (b *batch) publishBatch(ctx context.Context) (btcutil.Amount, error) {
 		batchTx.TxIn[i].Witness = witness
 	}
 
-	b.log.Debugf("attempting to publish non-coop tx with feerate=%v, "+
-		"totalfee=%v, sweeps=%v, destAddr=%s", b.rbfCache.FeeRate, fee,
-		len(batchTx.TxIn), address.String())
+	b.log.Infof("attempting to publish non-coop tx=%v with feerate=%v, "+
+		"totalfee=%v, sweeps=%v, destAddr=%s", batchTx.TxHash(),
+		b.rbfCache.FeeRate, fee, len(batchTx.TxIn), address.String())
```

Comment:
`.String()` call can be omitted (`%s` instructs printf-like functions to call it).

Also in another instance below in the PR.

Reply:
done